### ISSUE TITLE
Bash script not working

### ISSUE THREAD

**Issue by aparajey** [sipcapture/homer-installer#55]
Hello,

i tried to install homer with bash script on clean centos 7, but getting this error.

Complete!
[root@homer src]# nano homer_installer.sh
[root@homer src]# ./homer_installer.sh

[root@homer src]# **************************************************************
                                                              
      ,;;;;;,       HOMER SIP CAPTURE (http://sipcapture.org) 
     ;;;;;;;;;.     Single-Node Auto-Installer (beta 7.0)
   ;;;;;;;;;;;;;                                              
  ;;;;  ;;;  ;;;;   <--------------- INVITE ---------------   
  ;;;;  ;;;  ;;;;    --------------- 200 OK --------------->  
  ;;;;  ...  ;;;;                                             
  ;;;;       ;;;;   WARNING: This installer is intended for   
  ;;;;  ;;;  ;;;;   dedicated/vanilla OS setups without any   
  ,;;;  ;;;  ;;;;   customization and with default settings   
   ;;;;;;;;;;;;;                                              
    :;;;;;;;;;;     THIS SCRIPT IS PROVIDED AS-IS, USE AT     
     ^;;;;;;;^      YOUR *OWN* RISK, REVIEW LICENSE & DOCS    
                                                              
**************************************************************

ERROR: You need the following minimal set of commands installed:

       lsb_release wget curl dirmngr git

even though all of these required packages are installed but installer gives this error.
any help will be appreciated.

**Comment by aqsyonas** [comment]
@aparajey please pull the latest script and run it on freshly installed Centos7. Things should be working fine now.

**Comment by Justessej** [comment]
Where can i find the latest script do you have a wget link?

**Comment by aqsyonas** [comment]
@Justessej just clone the current homer-installer which contains latest code. 

`https://github.com/sipcapture/homer-installer.git`

**Comment by dv8inpp** [comment]
As this is a script issue I might point out that although it says Debian 9 the script referenced in the instructions on the front page is still set for Debian 8
wget https://cdn.rawgit.com/sipcapture/homer-installer/master/homer_installer.sh
but the acutal file https://github.com/sipcapture/homer-installer/blob/master/homer_installer.sh is correct

**Comment by lmangani** [comment]
Hi @dv8inpp You should use the 7.7 branch straight off Github to install the latest bundle

**Comment by Justessej** [comment]
Hi @lmangani i installed homer with the instructions on github with this link https://cdn.rawgit.com/sipcapture/homer-installer/master/homer_installer.sh. I get in the interface but when i reboot the system i cant login anymore is there a fix for this?

**Comment by lmangani** [comment]
@Justessej it depends on the version you have installed - is it 7.7?

**Comment by Justessej** [comment]
@lmangani Yes it is

**Comment by lmangani** [comment]
@Justessej great, please provide some logs errors, service status, etc to get some help

**Comment by Justessej** [comment]
Okay, I will upload some screenshots with the error

**Comment by Justessej** [comment]
![Screenshot_6](https://user-images.githubusercontent.com/57404162/72212773-0f788a80-34e3-11ea-804d-2114fecc6186.png)

**Comment by lmangani** [comment]
@Justessej the error is pretty clear. Postgres is down on your system. I would suggest checking that first.

**Comment by Justessej** [comment]
I installed Postgres and now i get a other error message 
![unknown (9)](https://user-images.githubusercontent.com/57404162/72224363-274f1d80-357a-11ea-9b8c-bb089ff93586.png)

**Comment by githubsesui** [comment]
> Hi @lmangani i installed homer with the instructions on github with this link https://cdn.rawgit.com/sipcapture/homer-installer/master/homer_installer.sh. I get in the interface but when i reboot the system i cant login anymore is there a fix for this?

I have the same issue. Can not login after reboot of the system when using the install script.

**Comment by Justessej** [comment]
@githubsesui Same here, waiting for a answer :)

**Comment by lmangani** [comment]
We're getting you all fixed fellas :)

Please install the latest `homer-app` I think it should resolve 100% of your issues. We just pushed it so feel free to open issues during installation for assistance. Once you installed the latest go `homer-app` please update your status

**Comment by Justessej** [comment]
![image](https://user-images.githubusercontent.com/57404162/72338749-6ccc3180-36c5-11ea-896b-feb8195dc3f7.png)

**Comment by Justessej** [comment]
Didnt see u need to manual setup some things im gonna try to setup this first

**Comment by Justessej** [comment]
@lmangani Is the manual setup for after running the homer installer script?

**Comment by githubsesui** [comment]
Hi Lorenzo,

Thank you for your reply.

Will the Bash Install Script be updated? I don’t have much experience with the manual installation and I am only using the Install Script.
Regards,

Philip

**Comment by Justessej** [comment]
@githubsesui Lets hope so the manual setup is to hard for me. Idk how to install any of this on my system.

**Comment by lmangani** [comment]
The installers are being updated - hold on tight, we'll make this as easy as it should be.

**Comment by aqsyonas** [comment]
Could you please, check the latest script and see if that works for you?

https://github.com/sipcapture/homer-installer.git

Br, Aqs.

**Comment by Justessej** [comment]
With the same way as:

cd /usr/src
wget https://cdn.rawgit.com/sipcapture/homer-installer/master/homer_installer.sh
chmod +x homer_installer.sh
./homer_installer.sh

**Comment by aqsyonas** [comment]
Yes, that looks good.

Br, Aqs.

**Comment by Justessej** [comment]
![Screenshot_3](https://user-images.githubusercontent.com/57404162/72605519-3be63980-391d-11ea-80af-0568894db7c6.png)
Nope still after reboot i cant login

**Comment by aqsyonas** [comment]
Which OS are you using? Could you check if you see some error in console
(press f12 and try to login again)

Br, Aqs.

**Comment by Justessej** [comment]
Im using CentOS 7 
![image](https://user-images.githubusercontent.com/57404162/72606096-4523d600-391e-11ea-9083-a0fa54d41a41.png)

**Comment by Justessej** [comment]
![image](https://user-images.githubusercontent.com/57404162/72606179-6be20c80-391e-11ea-806e-bfda1efe282d.png)

**Comment by aqsyonas** [comment]
could you please check a few things,

1. sestatus
2. systemctl status postgresql-12

Br, Aqs.

**Comment by Justessej** [comment]
![image](https://user-images.githubusercontent.com/57404162/72609440-32f96600-3925-11ea-94ce-2b3ea42c6af7.png)

**Comment by aqsyonas** [comment]
looks like your postgresq12 is dead.

Could you please run below command, and send me the output

systemctl restart postgresql-12
systemctl enable postgresql-12
systemctl status postgresql-12

Br, Aqs.

**Comment by Justessej** [comment]
![image](https://user-images.githubusercontent.com/57404162/72609702-e5c9c400-3925-11ea-9e01-5fe396a9db40.png)

**Comment by aqsyonas** [comment]
ok, great. Please, try to log in and see if it works this time. if not
check error in console.

Br, Aqs.

**Comment by Justessej** [comment]
![image](https://user-images.githubusercontent.com/57404162/72610522-e5322d00-3927-11ea-86e9-89acabe89552.png)
Still the same

**Comment by Justessej** [comment]
I DID IT!!!! Its working now homer-app was dead to so i did the same commands and when it was active i tried again and it worked but now i need help for connecting the company VoIP phones :)

**Comment by aqsyonas** [comment]
ok, great it worked for you. Please, open a separate thread for the other
help.

Br, Aqs.

**Comment by Justessej** [comment]
@aqsyonas I created the other issue thanks for your help <3

**Comment by githubsesui** [comment]
Hi Lorenzo,

I have used the newly updated installation script on Centos7 and it works. I also restarted the server and was able to login to Homer so the previous issue has been resolved.

Thank you for updating the script and resolving the issue.

Kind regards,

Philip

From: Philip Misra - Sesui
Sent: 16 January 2020 17:07
To: 'sipcapture/homer-installer' <reply@reply.github.com>; sipcapture/homer-installer <homer-installer@noreply.github.com>
Cc: Mention <mention@noreply.github.com>
Subject: RE: [sipcapture/homer-installer] Bash script not working (#55)

Hi Lorenzo,

Thank you for your reply.

Will the Bash Install Script be updated? I don’t have much experience with the manual installation and I am only using the Install Script.

Regards,

Philip
